blocks/recentreports/recentreports.css

```diff
@@ -0,0 +1,168 @@
+}
+
+.block.recentreports .box {
+  border: var(--line-width-hairline) solid var(--color-light-gray-dark);
+  border-radius: var(--space-8px);
+  padding: 1.25rem 1rem 4.06rem;
+  position: relative;
+  max-width: 16.5rem; 
+  height:100%;
+}
+
+.block.recentreports .discover-more {
+  text-align: center;
+  margin-top: 1rem;
+}
+
+.block.recentreports .discover-more .link-color {
+  font-size: var(--body-font-size-xlm);
+  line-height: var(--line-height-large);
+  font-weight: var(--font-weight-bold);
+  color: var(--link-color-orange);
+}
+
+.block.recentreports .discover-more .icon {
+  font-family: icomoon, sans-serif;
+  font-style: normal;
+  font-weight: var(--font-weight-medium);
+  font-variant: normal;
+  text-transform: none;
+  line-height: var(--space-1px);
+  display: inline-block;
+  vertical-align: middle;
+}
+
+.block.recentreports .discover-more .link-color .icon-up-arrow {
+  transform: rotate(90deg);
+  margin-left: var(--space-5px);
+}
+
+.block.recentreports .discover-more .link-color .icon-up-arrow::before {
+  content: "\e900";
+}
+
+.block.recentreports .box .row > div {
+  padding-top: var(--space-6px);
+  padding-bottom: var(--space-6px);
+}
+
+.block.recentreports .box .value-content label {
+  font-size: var(--heading-font-size-mxs);
+  line-height: var(--line-height-little-small);
+  color: var(--color-gray-light-text);
+}
+
+.block.recentreports .btn:not(.dropdown-toggle) {
+  color: var(--color-white);
+  font-family: var(--body-font-family);
+  text-transform: uppercase;
+  border-radius: 1.875rem;
+  padding: var(--space-5px) 1.25rem;
+  border: 1px solid transparent;
+ }
+
+.block.recentreports .box .box-footer {
+  position: absolute;
+  right: 0;
+  left: 0;
+  bottom: var(--space-15px);
+  text-align: center;
+  line-height: normal;
+}
+
+.block.recentreports .box .box-footer .btn {
+  font-size: var(--heading-font-size-ms);
+  min-height: 1.875rem;
+  font-weight: var(--font-weight-medium);
+  text-decoration: none;
+  background-color: var(--button-color-orange); 
+  display:inline-block;
+  line-height: var(--line-height-regular);
+}
+
+.block.recentreports .box .value-content .label-value {
+  font-size: var(--heading-font-size-m);
+  color: var(--color-gray-light-text);
+  font-weight: var(--font-weight-medium);
+  text-overflow: ellipsis;
+  overflow: hidden;
+}
+
+.block.recentreports .box .value-content .label-value.positive {
+  color: var(--color-green);
+}
+
+.block.recentreports .target-price-div,.ratings-div,.date-div {
+  position: relative;
+  width: 100%;
+  padding-right: var(--space-15px);
+  padding-left: var(--space-15px);
+  
+}
+
+.block.recentreports .target-price-div {
+  flex: 0 0 50%;
+  max-width: 50%;
+}
+
+.block.recentreports .ratings-div {
+  flex: 0 0 33.3333%;
+  max-width: 33.3333%;
+}
+
+.block.recentreports .date-div {
+  flex: 0 0 100%;
+  max-width: 100%;
+}
+
+main .section .block.recentreports .block.carousel.multi-slide .carousel-slide-indicators {
```

Comment:
let's use  `main .section` in front of other selectors also in this file.

Reply:
@shrotia do we really need main . section?  I was thinking of removing  if from here.